src/button/tests/unit/Button.ts

```diff
@@ -124,42 +125,42 @@ registerSuite({
 	},
 
 	events() {
-		let blurred = false;
-		let clicked = false;
-		let focused = false;
-		let keydown = false;
-		let keypress = false;
-		let keyup = false;
-		let mousedown = false;
-		let mouseup = false;
+		const onBlur = sinon.spy();
```

Comment:
I think that a `stub` would be more appropriate here as a `spy` is meant to allow the function to run underneath and simply monitor the args etc that it was called with.